### DISCUSSION TITLE
Filter events by `!=`, `neq`

### DISCUSSION THREAD

**Discussion by BobSquarePants** [evilsocket/opensnitch#1203] [Q&A]
Hi,
is it possible in the filter of the event tab. to filter by exclusion.
for example exclude all event that contain in their `cmd` 'foobar' ? like `!=` or a regex or whatever ?
![OS](https://github.com/user-attachments/assets/59988490-f557-401d-b2ae-59ad97a176e6)

Thanks.

**Comment by BobSquarePants** [comment]
little workaround in your rules you can tick `Don't log the connection` (in the **More** Tab )

This can be closed...

**Reply by gustavo-iniguez-goya** [reply]
right now it's not possible. There's a feature request already opened here: #1055

but we need to rewrite the main window before adding more features.
